ramalama/common.py

```diff
@@ -35,6 +35,14 @@
 
 MIN_VRAM_BYTES = 1073741824  # 1GiB
 
+SPLIT_MODEL_PATH_RE = r'(.*)/([^/]*)-00001-of-(\d{5})\.gguf'
```

Comment:
I do hope this format is somewhat strictly enforced/used, lets try it 😄

Reply:
The regex is from this PR from @afazekas introducing a big model split: https://github.com/containers/ramalama/pull/1426
I only moved it out of the factory to the URL model class (these are essentially just multiple files for one model). I haven't used it before, so would be great if it could be verified I didn't break anything.